### PR TITLE
Add Default & Highlight Cards

### PR DESCRIPTION
**Pull request by dzwood**
# Update
* added two new cards, `DefaultCard` and `HighlightCard`, based off of the `Cards` component

### CODE REVIEW

File: ui-kit/Card/Card.stories.js [hunk 1]
```diff
@@ -1,12 +1,14 @@
 import React from 'react';
 
-import Card from './Card';
+import Card from './';
```
Comment (drewbarontini) [inline ui-kit/Card/Card.stories.js:3]: This would go back to `./Card` if we make the change requested below.

**Review by drewbarontini — CHANGES_REQUESTED** [review]
Looking good, @dzwood! A few suggestions on this…

File: ui-kit/Card/Card.styles.js [hunk 2]
```diff
@@ -90,6 +90,7 @@ const Cover = styled.div`
   background-image: url(${props => props.src});
   background-repeat: no-repeat;
   background-size: cover;
+  background-position: center;
```
[the remark this answers is not in the document]
Comment (dzwood) [reply ui-kit/Card/Card.styles.js:93]: Oh okay got it!

File: ui-kit/Card/index.js [hunk 3]
```diff
@@ -1,3 +1,3 @@
-import Card from './Card';
+import Card from './Card.styles';
```
[the remark this answers is not in the document]
Comment (dzwood) [reply ui-kit/Card/index.js:1]: Yeah that was something @asleepingpanda had implemented, but I'm fine doing it that way.

**Comment by dzwood** [comment]
@drewbarontini I reimplemented to the `Styled` format for the cards so it matches the rest of the `ui-kit` and added `Card.js`. Also alphabetized those CSS properties, let me know if there's anything else! Thanks

File: ui-kit/HighlightCard/HighLightCard.styles.js [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+const HighlightCardLabel = styled.div`
+  background-color: rgba(255, 255, 255, .3);  
+  backdrop-filter: blur(7px);
+  display:inline-block;
+  color: ${themeGet('colors.white')};
+  font-size: ${themeGet('fontSizes.xs')};
+  font-weight: ${themeGet('fontWeights.bold')};
+  letter-spacing: 0.125rem;
+  padding: ${themeGet('space.s')} ${themeGet('space.base')};
+  border-radius: 1rem;
```
[the remark this answers is not in the document]
Comment (dzwood) [reply ui-kit/HighlightCard/HighLightCard.styles.js]: Got it, added some additional sizes to the `radii` in the `theme.js`.

**Review by drewbarontini — APPROVED** [review]
Looks good! 🎉

**Comment by drewbarontini** [comment]
@dzwood I'm not sure why the Netlify build is failing… 🤔

**Comment by dzwood** [comment]
> @dzwood I'm not sure why the Netlify build is failing… 🤔

It looks like it was the `EventLists`, forgot to update it to the new card component. It was causing the Events page to error out. Going to re deploy it now, hopefully should be it...

**Comment by dzwood** [comment]
@drewbarontini Deploy is now working, the issue was actually with the API and has now been fixed.

**Review by drewbarontini — APPROVED** [review]

**Pull request merged**